content/en/docs/architecture/release-gating.md

```diff
@@ -11,16 +11,18 @@ We in general have four types of jobs that control the movement of code to produ
 1. PR gating - a set of tests that block PR merges across the entire organization.  Each repository may run a different set of gating jobs (such as the installer repo testing multiple platforms).  Almost every repository will run a standard set of consistent tests to prevent regressions (component A breaks component B by changing an API)
 2. PR optional - a set of optional tests that can be invoked on PRs as needed by reviewers or testers to gain additional coverage, but which running all the time would be inefficient or unnecessary.
 3. Release gating - these jobs prevent the publication of a new release payload and are considered the minimum bar of quality before any combination of images is created.  These tests also catch regressions that PR gates do not.
+    * Being the last automated chance of catching a regression before a release is sent to our Quality Engineering organization, we have formed the Technical Release Team to resolve and prevent problems affecting release gating jobs.  As such, this team owns our release gating strategy.  All new release gating jobs must have their approval.
```

Comment:
My understanding was that TRT was in charge of the 4.dev branch (currently 4.10), but that the patch-manager was still in charge of released 4.y branches.  Although it's often TRT's tooling (Sippy, etc.) keeping an eye on both, and as TRT grows, I would not be surprised if they take a more active role in released 4.y job health.  Do we want to bring in the patch-manager here?  Or do we want to sweep that under the rug in these docs, and let folks looking for released-4.y blockers ask TRT, and have TRT forward them to the patch manager?

Reply:
I think it's fair to have a high level description of the TRT + Patch Manager involvement.  Let me rework this.

Reply:
I went a different direction based on @jluhrsen's comments.  I simplified this section to simply mention adding new release gating jobs are is the responsibility of the TRT.  There's now a whole new section explaining the TRT's mission and job aggregation.